src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1876,7 +1915,6 @@
       <Kind Name="CloseParenToken"/>
     </Field>
   </Node>
-
```

Comment:
It is not clear why the empty line was removed. Please keep it, it makes the file easier to read.

Reply:
Tracked in https://github.com/dotnet/roslyn/issues/23222